spec/amp-actions-and-events.md

```diff
@@ -61,7 +61,9 @@ Example: `on="submit-success:lightbox1;submit-error:lightbox2"`
 ## Globally defined Events and Actions
 Currently AMP defines `tap` event globally that you can listen to on any HTML element (including amp-elements).
 
-AMP also defines a `hide` action globally that you can trigger on any HTML element.
+AMP also defines `hide`, `show` and `toggle` actions globally that you can trigger on any HTML element.
+
+**Note:** An AMP element can only be shown if it was previously hidden by a `hide` or `toggle` action. Normal elements can set the `hidden` attribute if the element has to be hidden by default.
```

Comment:
I would reword as:
An element can only be shown if it was previously hidden by a `hide` or `toggle` action, or by using the [hidden](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden) attribute. `show` does **not** support elements hidden by CSS `display:none` or AMP's `layout=nodisplay`

@bpaduch for review.